core/src/main/resources/org/pentaho/platform/dataaccess/datasource/wizard/public/importDialog.properties

```diff
@@ -14,7 +14,7 @@ importDialog.VALUE=Value
 importDialog.LOC_BUNDLES=Localized Bundles
 importDialog.FILE_NAME=File Name
 importDialog.IMPORT_MONDRIAN=Import Analysis
-importDialog.IMPORT_METADATA=Import Metadata
+importDialog.IMPORT_METADATA=Import Metadata/Data Source Wizard
```

Comment:
Why are we changing this text here for importDialog.IMPORT_METADATA?

Reply:
Sathish asked to change this as a feedback